specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-01-15-preview/examples/DeleteFailoverGroup.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "sqlManagedInstanceName": "testSqlManagedInstance",
+    "failoverGroupName": "testFailoverGroupName",
+    "api-version": "2023-01-15-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
```

Comment:
>headers [](http://example.com/codeflow?start=7&length=7)

200 is for synchronous deletes. You dont need an azure-AsyncOperation header here

Reply:
Removed